docs/source/tutorials/SAR/SAR tutorial.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b3084a00",
+   "metadata": {},
+   "source": [
+    "<h1 style=\"font-size: 36px;\">SAR tutorial</h1>\n",
+    "\n",
+    "In this notebook, we introduce synergistic action representations (SAR) and demonstrate how to use these representations to facilitate the training of high-dimensional continuous control policies within MyoSuite.\n",
+    "\n",
+    "The primary purpose of this notebook is to equip the user to train with SAR for their own task specification(s) both within and beyond the musculoskeletal control paradigm.\n",
+    "\n",
+    "\n",
+    "# Table of contents\n",
+    "\n",
+    "- [Imports and utilities](#imports-and-utilities)\n",
```

Comment:
All of these 4 links don't work for me. I checked it using this link -- https://github.com/cameronberg/myosuite/blob/dev/docs/source/tutorials/SAR/SAR%20tutorial.ipynb

Reply:
links didn't seem to work on my end either so I removed them and updated table of contents accordingly